includes/Brands.php

```diff
@@ -103,7 +103,7 @@ public static function get_brands() {
 					'enabled_flows'  => array(
 						'ecommerce' => true,
 						'wp-setup'  => true,
-						'sitegen'   => true,
+						'sitegen'   => Config::has_ai_sitegen(),
```

Comment:
Do you think we can store this value once in a variable at the top and reuse it instead of calling the function multiple times?

Reply:
Sure